apps/nextjs-website/src/components/atoms/FileErrorIcon/FileErrorIcon.tsx

```diff
@@ -0,0 +1,42 @@
+import { SvgIcon } from '@mui/material';
+
+const FileErrorIcon = () => {
```

Comment:
Here I think we can safely use the `IllusError` icon coming from MUI-Italia.
https://github.com/pagopa/mui-italia/blob/main/src/illustrations/Error.tsx